packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,864 @@
+| onConflictingUser.action | String | Whether to proceed if more than one match is found. Currently only `DENY` is supported.                                                    |
+ 
+### Identifier Match Rule Object Example
+
+```json
+{
+    "name": "Catch-all Rule",
+    "id": "rul10y292nt87pGec0g4",
+    "type": "Okta:IdentifierMatch",
+    "priority": 0,
+    "conditions": [
+        {
+            "key": "Okta:UserType",
+            "op": "IN_LIST",
+            "value": [
+            	"${userTypeId}"
+        	]
+        },
+        {
+            "key": "Okta:Group",
+            "op": "INTERSECTS",
+            "value": [
+                "${groupId}"
+            ]
+        }	
+    ],
+    "action": "ALLOW",
+    "requirement": {
+        "identifyingAttributes": [
+            {
+                "userType": "otyt190o2hyJtSHVZ0g3",
+                "attribute": "email"
+            }
+        ],
+        "onConflictingUser": {
+            "action": "DENY"
+        }
+    },
+    "status": "ACTIVE",
+    "default": false,
+    "_links": {
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10y1rmKOjickDM0g4/rules/rul10y292nt87pGec0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT",
+                    "DELETE"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10y1rmKOjickDM0g4/rules/rul10y292nt87pGec0g4/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "policy": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10y1rmKOjickDM0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        }
+    }
+}
+```
+
+## Unknown User Policy Object
+
+The Unknown User Policy determines whether a user who has not been found to match with any existing Okta User Profiles is allowed to register and, if so, what User Type they should be assigned. One Unknown User Policy object is created by default. You cannot create additional Unknown User Policy objects.
+
+| Property | Type    | Description                                                                              |
+|----------|---------|------------------------------------------------------------------------------------------|
+| id       | String  | Unique identifier for this Policy (read-only).                                           |
+| name     | String  | Human-readable name for the Policy, configurable during creation or updating.            |
+| type     | String  | Type of the policy. For IdP Routing Policy objects, this needs to be `Okta:UnknownUser`. |
+| status   | String  | `ACTIVE`  or  `INACTIVE`.                                                                |
```

Comment:
Double whitespace twice between
```
`ACTIVE`  or  `INACTIVE`
```

Reply:
Fixed here and in a few other places where this was copy pasted.